File: tests/TestCase01.py
from lsst.ctrl.evmon import *

from lsst.ctrl.evmon.input import LsstEventReader
from lsst.ctrl.evmon.engine import MessageEvent

       
chain = Chain()

cond1 = Condition("$msg:info", Relation.STARTS_WITH, "Starting Stage")
chain.addLink(cond1)
        
cond2 = Condition("$msg:info", Relation.STARTS_WITH, "Ending Stage")
chain.addLink(cond2)
           
reader = LsstEventReader("LSSTLogging", "lsst8.ncsa.uiuc.edu")        
job = Job(reader, chain)

engine = EventMonitor(job)
engine.runJobs()

File: trunk/src/lsst/ctrl/evmon/test/TestCase01.java
package lsst.ctrl.evmon.test;

import lsst.ctrl.evmon.Chain;
import lsst.ctrl.evmon.Condition;
import lsst.ctrl.evmon.Job;
import lsst.ctrl.evmon.Relation;
import lsst.ctrl.evmon.input.LsstEventReader;


public class TestCase01 {

	public TestCase01() {

	}
	
	public Job createJob() {


		Chain chain = new Chain();
   	    chain.addLink(new Condition("$msg:info", Relation.STARTS_WITH, "Starting Stage"));
	
		chain.addLink(new Condition("$msg:info", Relation.STARTS_WITH, "Ending Stage"));
		
		LsstEventReader reader = new LsstEventReader("monitor");
		Job task = new Job(reader, chain);   	    
   	    return task;
	}
}
